README.md

```diff
@@ -1,59 +1,64 @@
 
-# Change to project directory
-cd LiU-AI-Project-Active-Learning-for-Music
+![learning-profile-examples](docs/datasets.png)
 
-# Create a virutal environment
-python -m venv venv
+Details about feature selection can be found under [Acquire and Build Datasets](#acquire-and-build-datasets) and details about priors and sliding window can be found under [Sliding Window](#sliding-window)
 
-# Activate the environment
-./venv/Scripts/Activate.ps1
+Note that by creating Learning Profiles that have different hyperparameters, but which are otherwise the same, the models are implicitly tuned.
 
-# Install pipwin and pyaudio (skip if you want pip to compile the binaries on its own)
-pip3 install pipwin==0.5.1
-pipwin install pyaudio==0.2.11
+A few examples of what a Learning Profile can look like is shown in the table below:
 
-# Install remaining packages
-pip3 install -r requirements.txt
-```
+![learning-profile-examples](docs/lp_examples.png)
 
-## FFMPEG Setup
 
-To allow different audio file formats the python package `pydub` is used. For playing mp3-files however, it is important that a valid mpeg-decoder is installed on the system. The `pydub` package defaults to using `libav` or `ffmpeg` if their binaries can be found on the system.
+## Sliding Window
+The ability to add a *sliding window* of a user defined size to a data set exists. This is in order to try to capture the chronological correlation of emotions in music, and it can be done in a few different ways. In the implementation a separation is made between how training and test data is configured with a sliding window. For training data, the labels from the training data set are used as indicators of emotion for the previous samples, while for test data a basic linear regressor is used to predict the emotion of previous samples in order to get an indication of the emotion. For both training data and test data a prior must be chosen, the prior will act as labels/predictions for the first samples of each song in the train/test dataset when configuring the sliding window. The options for training data are, one prior for all songs and no prior at all. When no prior is passed, the labels from the first few samples are used and those samples are then not used during the training. The options for test data are either one prior for all songs or a different prior for each song. Below is an illustration of how a sliding window size of *r* works on one song in the training data with a prior.
```

Comment:
"... the labels from the first few samples are used and those samples are then not used during the training"
Litet förvirrande mening.